parsers/IN_WE.py

```diff
@@ -67,7 +69,7 @@ def fetch_data(
 
     datetime_col = KIND_MAPPING[kind]["datetime_column"]
     for item in data:
-        item[datetime_col] = datetime.strptime(item[datetime_col], "%Y-%d-%m %H:%M:%S")
+        item[datetime_col] = datetime.strptime(item[datetime_col], "%Y-%m-%d %H:%M:%S")
```

Comment:
Yes need this to convert String datetime in 24h to python datetime

Reply:
please do use date time.fromisoformat() instead of pandas. We strive to use native date time handling as much as possible for performance and maintainability.